src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -46,6 +47,12 @@ def __init__(self, app, api, config, mount):
         # this need for the post validtiaon
         self.gq_service = WorkQueue(config.couch_host, config.couch_workqueue_db)
 
+        # setup ms-transferor URL using config (which is ConfigSection object)
+        if hasattr(config, "ms_transferor_url"):
+            self.msTransferorUrl = config.ms_transferor_url
```

Comment:
Do we need, any config related PR in parallel to this  one as well ?

Reply:
yes, I checked ReqMgr2 config.py and it does not have anything in data object to refer to cmsweb URL. Since we need to make HTTP request to MSTransferor we will need an additional PR for ReqMgr2 config.py to add cmsweb URL. I'll make it once we agree on current code changes.

Reply:
to adapt this I adjusted `test/python/WMCore_t/ReqMgr_t/TestConfig.py` codebase and added there new configuration parameter, otherwise lots of unit tests failed.